extjsdk/src/test/java/io/vantiq/extjsdk/TestUtils.java

```diff
@@ -57,9 +68,53 @@ public void testGetConfigInDir() throws Exception {
             Path p = Files.createFile(Paths.get("serverConfig/server.config"));
             f = new File(p.toString());
             f.deleteOnExit();
-            bw = fillProps(p);
+            bw = fillProps(p, true);
             
-            checkProps();
+            checkProps(false);
+        } finally {
+            if (bw != null) {
+                bw.close();
+            }
+            if (f != null) {
+                //noinspection ResultOfMethodCallIgnored
+                f.delete();
+            }
+            if (dir != null) {
+                //noinspection ResultOfMethodCallIgnored
+                dir.delete();
+            }
+        }
+    }
+
+    @Test
+    public void testGetEnvVar() throws Exception {
+        assumeTrue("\"CONNECTOR_AUTH_TOKEN\" environment variable must be set equal to \"xxxx====\"",
+                envVarAuthToken != null && envVarAuthToken.equals(FAKE_TOKEN));
+        doEnvVarTests(false);
+    }
+
+    @Test
+    public void testGetEnvVarOverwrite() throws Exception {
+        assumeTrue("\"CONNECTOR_AUTH_TOKEN\" environment variable must be set equal to \"xxxx====xxxx\"",
+                envVarAuthToken != null && envVarAuthToken.equals(FAKE_TOKEN2));
+        doEnvVarTests(true);
+    }
```

Comment:
Since the environment variable can only be set to one value, one of these tests will always be ignored. Both pass if their respective correct value is specified.